astropy/cosmology/io/mapping.py

```diff
@@ -216,7 +253,9 @@ def to_mapping(
     # Lastly, add the metadata, if haven't already (above)
     if not move_from_meta:
         m["meta"] = meta  # TODO? should meta be type(cls)
-
+    # Rename keys
+    if rename is not None:
+        m = _rename_map(m, rename)
     return m
```

Comment:
Equivalent outcome, but fewer lines of code:
```suggestion
    return m if rename is None else _rename_map(m, rename)
```